--- binutils_orig/config.sub	2015-09-19 13:54:19.160912923 +0300
+++ binutils-2.25/config.sub	2015-09-19 13:59:27.583710834 +0300
@@ -1384,7 +1384,8 @@
 	      | -os2* | -vos* | -palmos* | -uclinux* | -nucleus* \
 	      | -morphos* | -superux* | -rtmk* | -rtmk-nova* | -windiss* \
 	      | -powermax* | -dnix* | -nx6 | -nx7 | -sei* | -dragonfly* \
-	      | -skyos* | -haiku* | -rdos* | -toppers* | -drops* | -es* | -tirtos*)
+	      | -skyos* | -haiku* | -rdos* | -toppers* | -drops* | -es* | -tirtos* \
+	      | -tapios*)
 	# Remember, each alternative MUST END IN *, to match a version number.
 		;;
 	-qnx*)
